src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore/WrapPipelineMiddleware.cs

```diff
@@ -39,7 +39,7 @@ public async Task Invoke(HttpContext context)
             {
                 _agent.Logger.Log(Agent.Extensions.Logging.Level.Finest, "Skipping instrumenting incoming OPTIONS request.");
 
-                await _next(context);
+                await _next(context).ConfigureAwait(false);
```

Comment:
I don't know if this is safe to do. AspNetCore does not have an synchronization context by default, but if one was configured, I don't know if there would be problems with other middleware in the pipeline.